xbmc/guilib/GUIFontTTF.h

```diff
@@ -101,22 +116,24 @@ class CGUIFontTTF
     character_t glyphAndStyle;
     wchar_t letter;
   };
+
   struct RunInfo
   {
     int startOffset;
     int endOffset;
     hb_buffer_t* buffer;
     hb_script_t script;
-    hb_glyph_info_t* glyphInfos;
+    hb_glyph_info_t* glyphInfo;
+    hb_glyph_position_t* glyphPosition;
```

Comment:
I recommend naming  `glyphInfos` instead of `glyphInfo` and `glyphPositions` instead of `glyphPosition` as these are arrays